jdaviz/configs/imviz/tests/test_subset_centroid.py

```diff
@@ -15,7 +15,7 @@ def test_centroiding(self):
         # and nothing should change.
         for data_label in ('fits_wcs[DATA]', 'gwcs[DATA]', 'no_wcs'):
             plg._obj.dataset_selected = data_label
-            plg._obj.set_center((2, 2), update=True)  # Move the Subset back first.
+            plg._obj.set_center(new_cen=(2, 2), update=True)  # Move the Subset back first.
```

Comment:
I don't understand this change. `new_cen` is positional, not keyword, argument.
```suggestion
            plg._obj.set_center((2, 2), update=True)  # Move the Subset back first.
```